projects/observability/src/shared/components/radar/tooltip/radial-data-lookup-strategy.ts

```diff
@@ -16,6 +16,7 @@ export class RadialDataLookupStrategy {
   public constructor(private readonly allSeries: RadarSeries[], radialAxisData: RadarAxisData[]) {
     this.radialBisector = bisector(axisData => axisData.axisRadian);
     this.radialAxisData = clone(radialAxisData);
+    this.radialAxisData = this.radialAxisData.sort(axisData => axisData.axisRadian);
```

Comment:
We can do this in line. Just doing the sorting after cloning.
`this.radialAxisData = clone(radialAxisData).sort(axisData => axisData.axisRadian)`